drivers/rke/rke_driver.go

```diff
@@ -516,7 +516,30 @@ func (d *Driver) ETCDRestore(ctx context.Context, clusterInfo *types.ClusterInfo
 
 	dialers, externalFlags := d.getFlags(rkeConfig, stateDir)
 
-	err = cmd.RestoreEtcdSnapshot(ctx, &rkeConfig, dialers, externalFlags, snapshotName)
+	APIURL, caCrt, clientCert, clientKey, certs, err := cmd.RestoreEtcdSnapshot(ctx, &rkeConfig, dialers, externalFlags, snapshotName)
+	certStr := ""
+
+	if err == nil {
```

Comment:
is the code below common to update/restore? any chance we can extract it to its own method to re-use?